beater/beater.go

```diff
@@ -394,6 +396,31 @@ func (s *serverRunner) run() error {
 	s.acker.Open()
 	pipeline := pipetool.WithACKer(s.pipeline, s.acker)
 
+	if eac := os.Getenv("ELASTIC_AGENT_CLOUD"); eac == "" && s.config.JavaAttacherConfig.Enabled {
```

Comment:
@axw @simitt should I disable or delete this block of code so that we can merge the rest of the PR? there are still questions about other parts of the implementation and it's lower priority, so I could see it being a while before someone gets back to this.

Reply:
Yes let's remove this wiring-up bit until the rest is complete please. (FWIW I'm not a fan of the ELASTIC_AGENT_CLOUD check. We should probably just rely on the fact that seccomp will be enabled in ESS/ECE for security, and on the integration editor UI disallowing the Java attacher options.)